runhouse/servers/http/http_server.py

```diff
@@ -554,6 +556,40 @@ async def _get_results_and_logs_generator(fut, run_name, serialization=None):
             for f in open_logfiles:
                 f.close()
 
+    @staticmethod
+    @app.get("/processes")
+    @validate_cluster_access
+    async def get_processes(request: Request):
+        try:
+            processes = await obj_store.aget_all_initialized_servlet_names()
```

Comment:
i know not specifically part of this PR, but I think `aget_all_initialized_servlet_names` should return a set and not a list?

Reply:
does it matter much? there's no way to end up with duplicates